crawl-ref/source/item-prop-enum.h

```diff
@@ -194,6 +194,7 @@ enum jewellery_type
     RING_FIRST_RING = RING_PROTECTION,
     RING_PROTECTION_FROM_FIRE,
     RING_POISON_RESISTANCE,
+	RING_ELEC_RESISTANCE,
     RING_PROTECTION_FROM_COLD,
```

Comment:
`#pragma` tags are not indented in the crawl codebase so this is correct.

Reply:
This was referring to something very old, when this used tabs instead of spaces and `RING_ELEC_RESISTANCE` was out of alignment. Not sure why it popped up again.